### ISSUE TITLE
Double locales in fixtures?

### ISSUE THREAD

**Issue by aborhovich** [Sylius/Sylius#2800]
Having following exception on install:
```Please enter list of locale codes, separated by commas or just hit ENTER to use "en_US". For example "en_US, de_DE".
In which language your customers can browse the store?  
Adding English (United States).
PHP Fatal error:  Uncaught exception 'Doctrine\ORM\NonUniqueResultException' in /var/www/sylius/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php:623
Stack trace:
#0 /var/www/sylius/src/Sylius/Bundle/ResourceBundle/Doctrine/ORM/EntityRepository.php(78): Doctrine\ORM\AbstractQuery->getOneOrNullResult()
#1 /var/www/sylius/vendor/doctrine/orm/lib/Doctrine/ORM/EntityRepository.php(241): Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository->findOneBy(Array)
#2 /var/www/sylius/src/Sylius/Bundle/InstallerBundle/Command/SetupCommand.php(151): Doctrine\ORM\EntityRepository->__call('findOneByCode', Array)
#3 /var/www/sylius/src/Sylius/Bundle/InstallerBundle/Command/SetupCommand.php(151): Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository->findOneByCode('en_US')
#4 /var/www/sylius/src/Sylius/Bundle/InstallerBundle/Command/SetupCommand.php(48): Sylius\Bundle\InstallerBundle\Command\SetupCommand->setupLocales(Object(Symfony\Component\Console\Input\ArrayInput), Object(Symfony\Comp in /var/www/sylius/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php on line 623```

The source of problem is that locales duplicated in DB during installation: https://api.monosnap.com/rpc/file/download?id=q8TU2IvcuG2EEqOEbnluk7zKcLskmE

**Comment by kachar** [comment]
:+1: Removing the extra `en_US` record in table `sylius_locale` solved the problem

**Comment by pjedrzejewski** [comment]
Should be fixed by #2852.
